test/reductions/exponentiated_gradient/test_expgrad_smoke.py

```diff
@@ -2,26 +2,21 @@
 # Licensed under the MIT License.
 
 import pandas as pd
+from fairlearn.reductions import ExponentiatedGradient
 import fairlearn.reductions.moments as moments
-from fairlearn.reductions import exponentiated_gradient
 
 from test import simple_learners
+from .test_utilities import aux_data, X1, X2, X3, labels
 
 import pytest
 
 
 class TestExpgradSmoke:
     def setup_method(self, method):
         print("setup_method      method:%s" % method.__name__)
-        attrs = [str(x) for x in 'AAAAAAA' 'BBBBBBB' 'CCCCCC']
```

Comment:
Is there a specific reason why this has been moved into its own file? I don't care too much either way, but I'd like to stem proliferation of files where possible.

Reply:
moved it into one file